fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -59,6 +65,24 @@ TEST_P(Int32LogicalUnary, UsingConst) {
     ASSERT_EQ(exp, act);
 }
 
+TEST_P(Int64LogicalUnary, UsingConst) {
```

Comment:
Seems like there should also be a `UsingLoadParam` variant of this test that loads a parameter instead of loading a constant. In fact there should also be one for `ineg` (not sure why it's missing in the first place).